proxy/ControlBase.cc

```diff
@@ -459,8 +458,7 @@ MultiTextMod::set(char *value)
   Tokenizer rangeTok(",");
   int num_tok = rangeTok.Initialize(value, SHARE_TOKS);
   for (int i = 0; i < num_tok; i++) {
-    ts::Buffer text;
-    text.set(ats_strdup(rangeTok[i]), strlen(rangeTok[i]));
+    std::string_view text = std::string_view(ats_strdup(rangeTok[i]), strlen(rangeTok[i]));
```

Comment:
This is an excellent case for using `TextView` instead of `string_view` so the parsing can be done without the need for `Tokenizer` and the attendant memory allocation.